app/src/main/java/com/duycuong/weather/widget/dialog/progress/DialogManager.java

```diff
@@ -0,0 +1,33 @@
+package com.duycuong.weather.widget.dialog.progress;
+
+import android.app.ProgressDialog;
+import android.content.Context;
+
+import com.duycuong.weather.R;

```

Comment:
ERROR: Unused import: com.duycuong.weather.R. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports